<launch>
    <param name="~autorepeat_rate" type="double" value="10.0"/>
    <node pkg="joy" type="joy_node" name="joy_node" />
    <node pkg="robot_teleop" type="joy_to_ackermann.py" name="joy_to_ackermann"/>
        <rosparam file="$(find robot_teleop)/config/joy_ackermann.yaml" />
    <node pkg="robot_teleop" type="ackermann_controller.py" name="ackermann_controller"/>
        <rosparam file="$(find robot_teleop)/config/ackermann_controller.yaml" />
</launch>